src/functions.jl

```diff
@@ -1,47 +1,38 @@
 using DataFrames
 """
-    npt_running_average(x_val, y_val, n = 5, x_name = "x", y_name = "y")
+    npt_running_average(x, y, n = 5, x_name = "x", y_name = "y")
     
-# Defines a running average function
-#####################################################################################################
-# Input:                                                                                            #
-# x_val: x-values in the data-set which will be averaged.                                           #
-# y_val: y-values in the data-set which will be averaged.                                           #
-# n: Default 5. Defines how long the running average will be.                                       #
-# x_name: Default "x". Defines the name of the x-value column in the output DataFrame.              #
-# y_name: Default "y". Defines the name of the y-value column in the output DataFrame.              #
-#                                                                                                   #
-# Output:                                                                                           #
-# df: DataFrame of the averaged y-values and respective x values with given or default column-names #
-#####################################################################################################
+Calculates an `n`-point running average for vectors `x` and `y`.
+
+Returns a two-column DataFrame containing the running average values. `x_name` and `y_name` 
```

Comment:
`DataFrame` (with backticks), or just "dataframe"/"data frame" (whatever the convention in the Dataframes.jl package is).